app/Config/Routing.php

```diff
@@ -106,7 +108,7 @@ class Routing extends BaseRouting
      *       'blog' => 'Acme\Blog\Controllers',
      *   ]
      *
-     * @var array [ uri_segment => namespace ]
+     * @var array<string, string> [ uri_segment => namespace ]
```

Comment:
The example(E.g.) is clear to me, I think `[ uri_segment => namespace ] ` is not needed.

Reply:
Removed.